data/docs/gcp-monitoring/gke-tracing.mdx

```diff
@@ -0,0 +1,1190 @@
+
+You can refer to this [GitHub repository](https://github.com/isItObservable/pixie/tree/master/pixie%20plugin) for the different custom scripts for the different use cases.
+
+Use the appropriate export URL in the script. The export URL will receive all the traces captured by Pixie.
+
+```sh
+url='otel-collector.kubelet-otel.svc.cluster.local:4317',
+insecure=True
+```
+
+<figure data-zoomable align="left">
+    <img
+        src="/img/docs/gcp-monitoring/gke-tracing/pixie-config-cloud.webp"
+        alt="Pixie Configuration"
+    />
+<figcaption>
+<i>
+Pixie Configuration
+</i>
+</figcaption>
+</figure>
+
+
+**Step 3:** To test the configuration, deploy a sample application in the cluster to view its traces in the SigNoz.
+
+```sh
+kubectl run -n signoz my-hello –image=nginx –port=80
+```
+
+### Visualize the Traces in SigNoz Cloud
+
+To visualize the traces log into the SigNoz account and navigate to the traces section.
+
+<figure data-zoomable align="left">
+    <img
+        src="/img/docs/gcp-monitoring/gke-tracing/signoz-cloud-ebpf.webp"
+        alt="Traces in SigNoz Dashboard"
+    />
+<figcaption>
+<i>
+Traces in SigNoz Dashboard
+</i>
+</figcaption>
+</figure>
+
+## APM and Distributed Tracing
+
+For application-level tracing, you can use the OpenTelemetry SDKs integrated with your application. These SDKs automatically collect and forward traces to the central collector.
+
+Please refer to our [SigNoz Tutorials](https://signoz.io/docs/instrumentation/) or [Blog](https://signoz.io/blog/) to find information on how to instrument your application like Spring, FastAPI, NextJS, Langchain, Node.js, Flask, Django, etc.
+
+## Sample Python Application
+
+We will use a [sample flask app](https://github.com/SigNoz/sample-flask-app).
+One may need to add packages for enabling instrumentation.
+
+requirements.txt
+
+```txt
+Flask==3.0.0 
+pymongo==3.12.1 
+requests==2.26.0
+opentelemetry-api==1.22.0 
+opentelemetry-distro==0.43b0 
+opentelemetry-instrumentation==0.43b0
+opentelemetry-exporter-otlp==1.22.0
+```
+
+Install those packages in dockerfile configuration.
+
+```dockerfile
+…
+
+# install dependencies
+RUN pip install -r requirements.txt
+RUN opentelemetry-bootstrap --action=install
+…
+```
+
+### Example YAML
+
+For running this sample-flask-app as pod
+
+sample-flask-app.yaml
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  creationTimestamp: null
+  labels:
+    app: sample-flask-app
+  name: sample-flask-app
+spec:
+  containers:
+  - image: mongo:latest
+    name: mongo
+    ports:
+    - name: mongo
+      containerPort: 27017
+  - image: signoz/sample-flask-app:latest
+    name: sample-app
+    ports:
+    - name: flask
+      containerPort: 5002
+    env:
+    - name: MONGO_HOST
+      value: localhost
+    - name: OTEL_RESOURCE_ATTRIBUTES
+      value: service.name=sample-app
+    - name: OTEL_EXPORTER_OTLP_ENDPOINT
+      value: http://<app-fqdn>:4317
```

Comment:
[app-fqdn](http://kubelet-otel-k8s-infra-otel-agent.signoz.svc.cluster.local:4317/) it is hyperlink.
But here it is not.

Reply:
why it would be hyperlink I didn't get you?

Reply:
It is yaml right?

Reply:
fixed